### ISSUE TITLE
No function like probas_to_classes

### ISSUE THREAD

**Issue by vladimircape** [fchollet/keras#5961]
I tried
from keras.utils.np_utils import probas_to_classes
but didn't find it.
What is new functionality?

**Comment by kgrm** [comment]
`probas.argmax(axis=-1)`

**Comment by bis-carbon** [comment]
What is the replacement for  "probas_to_classes" in keras 2.0.3 ?

**Comment by koutsocheras99** [comment]
> What is the replacement for "probas_to_classes" in keras 2.0.3 ?

@bis-carbon 

According to [here](https://keras.io/utils/) it has changed to "to_categorical".

**Comment by nanswamy** [comment]
In case of  pred=model.predict(img)

I used (pred[0:])  to fetch the class label  for the decision making 

I case of Binary class prediction (pred[0:])  results out "1" or "0"
